apps/ledger-live-desktop/src/renderer/hooks/useStoryly.tsx

```diff
@@ -0,0 +1,15 @@
+function useStoryly() {
+  const script = document.createElement("script");
+
+  script.src = "https://web-story.storyly.io/v2/storyly-web.js";
+  script["custom-element"] = "storyly-web";
+  script.async = true;
+
+  document.body.appendChild(script);
+
+  return () => {
+    document.body.removeChild(script);
+  };
```

Comment:
I see that this hook is used in the "root" component (Default)

Do we want it this function to be called once this root component is mounted ?
If yes, you should put your code in a `useEffect`. And put your cleaning function (the one in the return) inside the `useEffect`

With a `[]` empty dependency array, this function (actually right now it's not a hook technically, without any `useState` or `useEffect`) will be run and clean only once (on mount and unmount)

Reply:
^ this